server/cmd/main.go

```diff
@@ -273,6 +276,33 @@ func main() {
 	connToInstanceTracker := handlers.ConnectionToStateMachineInstanceTracker{
 		ConnectToInstanceMap: make(map[uuid.UUID]*machines.StateMachine, 0),
 	}
+	//lazy loading for more Registrants to Register their entities like Istio
+	time.AfterFunc(15*time.Second, func() {
```

Comment:
No timeouts of any length should be used. Use different logic, please.

Reply:
Do we need to log the Adapter in the boot-time log ?
while we are logging the Registrants whose entities are being imported the artifacthub comes first and registers but if we have any other adapter running locally then it pings the endpoint
/api/meshmodel/components/register
so how do I log this because it's pinging the endpoint continously and after the registrant is registered it still takes few seconds to register other entites

Reply:
Closing the loop:

> Thanks for asking. No, instead a server log can be emitted at the point in time that a host registers using this API.
> 
> An event also needs to be recorded to the Notification Center.
> 
> I believe that there are instructions in the contributor guide for doing so (sending an event to the Notification Center).